media/examples/external.css

```diff
@@ -0,0 +1,3 @@
+p {
```

Comment:
I think it would be helpful to give this CSS file a more explicit name, like "link-element-example.css" or something.

Even consider including a comment in the file, too, like:
```
/*
This CSS file is used in the example for the <link> element.
*/
```

...that way someone stumbling across it will understand what it's for :)